src/actions/surveySubmission.js

```diff
@@ -8,7 +8,7 @@ export function retrieveSurveySubmissions(surveyId = null, linked = null, page =
         let promise;
 
         if (linked !== null) {
-            linked = linked? ((filters.linked == 'linked')? 1 : 0) : null;
+            linked = linked? ((linked == 'linked')? 1 : 0) : null;
```

Comment:
As a formatting comment, when I write ternary conditional statements, I usually place the question mark whitespace-separated. 

The hope then is that the reader can separate the parts of the statement.

Reply:
I agree with this, but totally missed it when fixing this issue. Thanks for pointing it out! Will commit change asap :)